Documentation/help_files/cms_1500_help.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * CMS 1500 Help.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @version 1.0.0
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ 
+use OpenEMR\Core\Header;
```

Comment:
I know this is anal, but lets put the use statements below the require_once blocks

Reply:
oh, this is the help script, disregard above comment

Reply:
Sure, will do. I understand anal -  being in the specialty that I am in :)